public/javascripts/Gallery/css/cards.css

```diff
@@ -1,75 +1,182 @@
 .gallery-card {
     position: relative;
     display: inline-block;
-    border-style: solid;
-    border-width: 1px;
-    padding: 5px;
-    border-radius: 5px;
-    margin: 2px;
-    width: 360px;
+    padding: 0px;
+    margin: 5px;
+    border-radius: 20px;
+    width: 22vw;
     height: auto;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.1), 0 6px 20px 0 rgba(0, 0, 0, 0.05);
+    grid-area: 'gallery-card';
+}
+.gallery-card:after {
+    content: '';
+    position: absolute;
+    z-index: -1;
+    width: 100%;
+    height: 100%;
+    border-radius: 20px;
+    opacity: 0;
+    bottom:0;
+    left:0;
+    box-shadow: 0 5px 15px rgba(0,0,0,0.3);
+    transition: opacity 0.3s ease-in-out;
+}
+.gallery-card:hover:after {
+    opacity: 1;
 }
 
 .static-gallery-image {
     width: 100%;
     height: auto;
+    border-top-left-radius: 20px;
+    border-top-right-radius: 20px;
 }
 
 #gallery-validation-button-holder {
     display: flex;
     flex-flow: row wrap;
     justify-content: center;
-    position: absolute;
-    bottom: 50px;
-    left: 5px;
-    right: 5px;
-    height: 11.5%;
+    position: relative;
+    /* bottom: -11.5%; */
+    left:0;
+    right:0;
+    height: 20%;
+    color: black;
+    font-size: 0.75vw;
+    font-weight: bold;
+    z-index: 2;
 }
 
 #gallery-card-agree-button {
-    background-color: green;
     width: 33%;
+    border-bottom-left-radius: 20px;
+    border-right: 1px solid #dcdbdb;
 }
 
 #gallery-card-disagree-button {
-    background-color: red;
     width: 34%;
+
 }
 
 #gallery-card-not-sure-button {
-    background-color:orange;
+    border-left: 1px solid #dcdbdb;
     width: 33%;
+    border-bottom-right-radius: 20px;
 }
 
-.label-icon {
+.card-header {
+    display: flex;
     position: absolute;
-    width: 12px;
-    z-index: 2;
+    left: 1vw;
+    top: 1vh;
+    font-family: 'raleway-regular';
```

Comment:
I think in general you're supposed to put a backup generic font-family like `sans-serif` after

Reply:
this is still missing in a lot of places

Reply:
ahh wait I'll just fix it now